test/test_log.ts

```diff
@@ -0,0 +1,77 @@
+import * as Chai from "chai";
+import * as ChaiAsPromised from "chai-as-promised";
+import * as Proxyquire from "proxyquire";
+import * as RealLog from "../src/log";
+Chai.use(ChaiAsPromised);
+const expect = Chai.expect;
+
+let createdLogger = null;
+let loggerClosed = false;
+let loggedMessages = [];
+
+const WinstonMock = {
+    createLogger: (format, transports) => {
+        return createdLogger = {
+            format,
+            transports,
+            close: () => {
+                loggerClosed = true;
+            },
+            silent: false,
+            log: (type, ...msg) => {
+                loggedMessages = loggedMessages.concat(msg);
+            },
+        };
+    },
+};
+
+const Log = (Proxyquire("../src/log", {
+    winston: WinstonMock,
+}).Log);
+
+describe("Log", () => {
+
+    beforeEach(() => {
+        loggerClosed = false;
+        loggedMessages = [];
+    });
+
+    describe("ConfigureBridge", () => {
+        it("should pass if config is empty", () => {
+            Log.ConfigureBridge({});
```

Comment:
This has no checks, shouldn't you do something like try...catch to expect it to pass?

Reply:
It should fail if it throws. We can't really test the config because the config is just the default which might change.